### PR TITLE
[bitnami/nginx] Added https port to the container via containerTlsPor…

### PR DESCRIPTION
**Pull request by fsniper**
…t option

This changes adds containerTlsPort option to enable nginx to serve https

**Benefits**

This benefits users who would likte to terminate tls at nginx.

**Possible drawbacks**

**Applicable issues**

  - fixes #

**Additional information**

**Checklist** 
- [X] Chart version bumped in `Chart.yaml` according to [semver](http://semver.org/).
- [X] Variables are documented in the README.md
- [X] Title of the PR starts with chart name (e.g. `[bitnami/chart]`)
- [X] If the chart contains a `values-production.yaml` apart from `values.yaml`, ensure that you implement the changes in both files

:warning: Keep in mind that if you want to make changes to the kubeapps chart, please implement them in the [kubeapps repository](https://github.com/kubeapps/kubeapps/tree/master/chart/kubeapps). This is only a synchronized mirror.

### CODE REVIEW

File: bitnami/nginx/README.md [hunk 1]
```diff
@@ -77,6 +77,7 @@ The following tables lists the configurable parameters of the NGINX Open Source
 | `extraVolumeMounts`                        | Array of extra volume mounts to be added to the deployment. Normally used with `extraVolumes`| `nil`                                                        |
 | `replicaCount`                             | Number of replicas to deploy                                                                 | `1`                                                          |
 | `containerPort`                            | Deployment Container Port                                                                    | `8080`                                                       |
+| `containerTlsPort`                         | Optional Deployment Container Tls Port                                                       | `nil`                                                        |
```
Comment (carrodher) [inline bitnami/nginx/README.md:80]: I would remove the "Optional" here as we are describing the purpose of the parameter, if it's optional or not it depends on the configuration, by default is not enabled.

File: bitnami/nginx/Chart.yaml [hunk 2]
```diff
@@ -1,6 +1,6 @@
 apiVersion: v1
 name: nginx
-version: 5.6.1
+version: 5.7.1
```
Comment (carrodher) [inline bitnami/nginx/Chart.yaml:3]:
```suggestion
version: 5.7.0
```

[the remark this answers is not in the document]
Comment (fsniper) [reply bitnami/nginx/Chart.yaml]: Bummer. I always do this mistake. Fixed.

**Comment by carrodher** [comment]
Thanks, I am going to launch it in our CI/CD system, is everything works as expected the chart should be released

**Pull request merged**